Patch plonebrowser for styling issues.

bierik opened the pull request:
Fixes 4teamwork/bern.web#188

For patching the iframe in the plonebrowser a link tag injection with
javascript is nessesary. At the moment the patch works only on overlay
load event.

![bildschirmfoto 2015-07-10 um 16 57 19](https://cloud.githubusercontent.com/assets/1637820/8621268/ff4ea408-2724-11e5-84ae-e3ea3efc1979.png)
![bildschirmfoto 2015-07-10 um 16 57 38](https://cloud.githubusercontent.com/assets/1637820/8621266/ff4deafe-2724-11e5-85af-4588d47a66d0.png)
![bildschirmfoto 2015-07-10 um 16 57 49](https://cloud.githubusercontent.com/assets/1637820/8621271/ff4f80c6-2724-11e5-8962-841e152b1cbd.png)
![bildschirmfoto 2015-07-10 um 16 57 57](https://cloud.githubusercontent.com/assets/1637820/8621267/ff4e787a-2724-11e5-8eda-a495b0b90633.png)
![bildschirmfoto 2015-07-10 um 16 58 14](https://cloud.githubusercontent.com/assets/1637820/8621269/ff4fb974-2724-11e5-81cf-6f8418404ab0.png)
![bildschirmfoto 2015-07-10 um 16 58 22](https://cloud.githubusercontent.com/assets/1637820/8621270/ff4fb4f6-2724-11e5-8dc7-36e85b797fc1.png)
![bildschirmfoto 2015-07-10 um 16 58 30](https://cloud.githubusercontent.com/assets/1637820/8621272/ff6a6b02-2724-11e5-87e2-e8eb609a6cf2.png)
![bildschirmfoto 2015-07-10 um 16 58 38](https://cloud.githubusercontent.com/assets/1637820/8621273/ff6afa7c-2724-11e5-9db4-ace913869812.png)

Commit:
Patch plonebrowser for styling issues.

Fixes 4teamwork/bern.web#188

For patching the iframe in the plonebrowser a link tag injection with
javascript is nessesary. At the moment the patch works only on overlay
load event.

jone commented on plonetheme/onegovbear/theme/javascript/tinyMCEIntegration.js:
```diff
@@ -0,0 +1,40 @@
+TinyMCE Dialog is loaded in an iFrame so it's not possible
+to override the stlying. So this script copies the related
+stylings to the head section of the iframe.
+TODO: Not working on page load.
```
Any idea how we can add support for non-overlay instances of tinymce?
Is there an event fired after tinymce is initialized, maybe? E.g. in Plone's TinyMCE integration?

jone commented:
Please rebase and it would be great to have some screenshots :wink:

bierik commented:
@jone updated

jone commented:
When the pathbar is too long, it doesn't break nicely
<img width="1279" alt="bildschirmfoto 2015-07-10 um 17 06 04" src="https://cloud.githubusercontent.com/assets/7469/8621494/2c950460-2726-11e5-8ff3-b7e3a7ff20e9.png">

jone commented:
The padding isn't the same for all tabs:
<img width="304" alt="bildschirmfoto 2015-07-10 um 17 08 24" src="https://cloud.githubusercontent.com/assets/7469/8621532/5c77994a-2726-11e5-8f39-a96b12354a75.png">
<img width="320" alt="bildschirmfoto 2015-07-10 um 17 08 29" src="https://cloud.githubusercontent.com/assets/7469/8621531/5c75ccf0-2726-11e5-95dc-a768647dd637.png">

jone commented:
<img width="1256" alt="bildschirmfoto 2015-07-10 um 17 10 29" src="https://cloud.githubusercontent.com/assets/7469/8621564/96cdfe22-2726-11e5-8317-320d0c659695.png">

jone commented:
<img width="1050" alt="bildschirmfoto 2015-07-10 um 17 11 14" src="https://cloud.githubusercontent.com/assets/7469/8621578/b260a89c-2726-11e5-9d04-b927ddfd12ff.png">
<img width="1129" alt="bildschirmfoto 2015-07-10 um 17 11 04" src="https://cloud.githubusercontent.com/assets/7469/8621577/b260bb98-2726-11e5-80d7-43e23e8d1d2b.png">

jone commented:
If we update tinymce (`Products.TinyMCE = 1.3.10`) it will not work anymore.
@maethu how do we want to handle this?

bierik commented:
@jone updated

![bildschirmfoto 2015-07-13 um 09 39 23](https://cloud.githubusercontent.com/assets/1637820/8645680/b8993e70-2949-11e5-9f89-4d8aa289867d.png)
![bildschirmfoto 2015-07-13 um 10 14 25](https://cloud.githubusercontent.com/assets/1637820/8645676/b8959d7e-2949-11e5-89e1-aa6d50cb93fb.png)
![bildschirmfoto 2015-07-13 um 10 14 41](https://cloud.githubusercontent.com/assets/1637820/8645677/b896b222-2949-11e5-98e6-5ce6c8bfdf2f.png)
![bildschirmfoto 2015-07-13 um 10 14 56](https://cloud.githubusercontent.com/assets/1637820/8645678/b8981b62-2949-11e5-8ac3-d7fcbde2666c.png)
![bildschirmfoto 2015-07-13 um 10 26 57](https://cloud.githubusercontent.com/assets/1637820/8645679/b898be32-2949-11e5-8ffa-fb0a27ebad72.png)

Commit:
Patch onOpen._add function in tinyMCE.

Because the update to 1.3.10 the javascript throws errors when attaching
an event handler to ```onOpen``` because the ```_add``` function is not
in the scope of Event but the ```add``` function. So the best
workaround it to bind the ```_add``` function to ```add```.i
See https://github.com/4teamwork/bern.web/pull/206 for update
information.

bierik commented:
@jone

Depends on 4teamwork/bern.web#206

jone commented on plonetheme/onegovbear/theme/javascript/tinyMCEIntegration.js:
```diff
@@ -0,0 +1,45 @@
+            editor.windowManager.onOpen.add(function() {
+              if (this.lastId) {
+                /*
+                  _add in not in scope of Event.
```
typo?
